management/prompt.go

```diff
@@ -30,3 +36,39 @@ func (m *PromptManager) Read(opts ...RequestOption) (p *Prompt, err error) {
 func (m *PromptManager) Update(p *Prompt, opts ...RequestOption) error {
 	return m.Request("PATCH", m.URI("prompts"), p, opts...)
 }
+
+// MarshalJSON is a custom serializer for the promptCustomText type.
+//
+// We have to use one to avoid creating a new Request method that does not JSON-encode the request body.
+func (t *promptCustomText) MarshalJSON() ([]byte, error) {
+	return []byte(auth0.StringValue(t.Text)), nil
+}
+
+// UnmarshalJSON is a custom deserializer for the promptCustomText type.
+//
+// We have to use one to avoid creating a new Request method that does not JSON-decode the response body.
+func (t *promptCustomText) UnmarshalJSON(b []byte) error {
+	t.Text = auth0.String(string(b))
+	return nil
+}
+
+// Retrieve custom text for a specific prompt and language.
```

Comment:
Comment should have the following format 'CustomText ...' (with an optional leading article) 

From: https://golang.org/doc/effective_go#commentary

Reply:
That's correct. Some methods have the corresponding docs page description, though: https://github.com/go-auth0/auth0/blob/master/management/branding.go#L149.
I'll update it, thanks!

Reply:
Changed in https://github.com/go-auth0/auth0/pull/240/commits/1016644b84f927ba9fc2741a649cee693e808dad